src/main/java/core/basesyntax/DrawAble.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface DrawAble {
```

Comment:
Drawable is a single word 
```suggestion
public interface Drawable {
```